motion/p5js/raindrops/index.js

```diff
@@ -0,0 +1,131 @@
+// Circle1
+var circleX;
+var circleY;
+
+// Circle2
+var circleX2;
+var circleY2;
+
+
+// Circle3
+var circleX3;
+var circleY3;
+
+
+// Circle4
+var circleX4;
+var circleY4;
+
+
+// Circle5
+var circleX5;
+var circleY5;
+
+var d1 = 8;
+
+//Rectangle 
+
+var rectX;
+var rectH;
+
+
+function setup() {
+   createCanvas(500,500);
```

Comment:
assigned to the sketch id div https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-P5js-Sketches#code